src/QueryLanguage/Processor/Doctrine/Dbal/Column.php

```diff
@@ -0,0 +1,75 @@
+<?php declare(strict_types=1);
+
+namespace Fazland\ApiPlatformBundle\QueryLanguage\Processor\Doctrine\Dbal;
+
+use Doctrine\DBAL\Query\QueryBuilder;
+use Doctrine\DBAL\Types\Types;
+use Fazland\ApiPlatformBundle\QueryLanguage\Expression\ExpressionInterface;
+use Fazland\ApiPlatformBundle\QueryLanguage\Processor\ColumnInterface;
+use Fazland\ApiPlatformBundle\QueryLanguage\Walker\Dbal\SqlWalker;
+
+/**
+ * @internal
+ */
+class Column implements ColumnInterface
+{
+    private string $columnType;
```

Comment:
as you're in the Column class you can omit the `column` prefix, what do you think?

Reply:
`column` prefix was also used in `ORM\Column` class, but as already said, probably the "Column" name was a bad choice.
We need to think about an alternative (Field ?), but is out of the scope of this PR. I'm leaving it as it is, until a decision about the naming is taken, then we can rename this and the ORM's one.

Reply:
Totally agree 😄